tracer/src/Datadog.Trace/Logging/Internal/Configuration/LogSinkOptions.cs

```diff
@@ -10,4 +10,5 @@ namespace Datadog.Trace.Logging.Internal.Configuration;
 internal static class LogSinkOptions
 {
     public const string File = "file";
+    public const string Console = "console";
```

Comment:
Do we want to call this `console-experimental` or something for now and merge this?

Reply:
yes, and done